baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/InvocationHandlerDelegation.java

```diff
@@ -47,12 +47,13 @@
         link = "https://github.com/palantir/gradle-baseline#baseline-error-prone-checks",
         linkType = BugPattern.LinkType.CUSTOM,
         severity = BugPattern.SeverityLevel.WARNING,
-        summary = "InvocationHandlers which delegate to another object must catch and unwrap "
-                + "InvocationTargetException, otherwise an UndeclaredThrowableException will be thrown "
-                + "each time the delegate throws an exception.\n"
-                + "This check is intended to be advisory. It's fine to "
-                + "@SuppressWarnings(\"InvocationHandlerDelegation\") in certain cases, "
-                + "but is usually not recommended.")
+        summary =
+                "InvocationHandlers which delegate to another object must catch and unwrap "
+                        + "InvocationTargetException, otherwise an UndeclaredThrowableException will be thrown "
+                        + "each time the delegate throws an exception.\n"
+                        + "This check is intended to be advisory. It's fine to "
+                        + "@SuppressWarnings(\"InvocationHandlerDelegation\") in certain cases, "
+                        + "but is usually not recommended.")
```

Comment:
Bit of a shame about these multi-line strings in annotation parameters - they're not a deal breaker but just make the diff noisier